src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -1741,12 +1741,14 @@ def test_network_subnet_delegation(self, resource_group):
         result = self.cmd('network vnet subnet list-available-delegations -g {rg}').get_output_in_json()
         self.assertTrue(len(result) > 1, True)
 
-        # self.cmd('network vnet create -g {rg} -n {vnet} -l westcentralus')
-        # self.cmd('network vnet subnet create -g {rg} --vnet-name {vnet} -n {subnet} --address-prefix 10.0.0.0/24 --delegations Microsoft.Sql/servers',
-        #         checks=self.check('delegations[0].serviceName', 'Microsoft.Sql/servers'))
-        # # verify the update command, and that CLI validation will accept either serviceName or Name
-        # self.cmd('network vnet subnet update -g {rg} --vnet-name {vnet} -n {subnet} --delegations Microsoft.Sql.Servers',
-        #         checks=self.check('delegations[0].serviceName', 'Microsoft.Sql/Servers'))
+        self.cmd('network vnet create -g {rg} -n {vnet} -l westcentralus')
+        self.cmd('network vnet subnet create -g {rg} --vnet-name {vnet} -n {subnet} --address-prefix 10.0.0.0/24 --delegations Microsoft.Sql/servers', checks=[
+            self.check('delegations[0].serviceName', 'Microsoft.Sql/servers'),
+            self.check('purpose', 'InterfaceEndpoints')
+        ])
+        # verify the update command, and that CLI validation will accept either serviceName or Name
+        self.cmd('network vnet subnet update -g {rg} --vnet-name {vnet} -n {subnet} --delegations Microsoft.Sql.Servers',
+                 checks=self.check('delegations[0].serviceName', 'Microsoft.Sql/Servers'))
 
```

Comment:
Are we able to add tests for the new interface-endpoint`` commands?

Reply:
I can test that they don't fail. They won't return anything because you can't create them.